package.json

```diff
@@ -31,96 +31,54 @@
-    "build:transpile": "babel src --out-dir lib --copy-files",
-    "lint": "yarn run lint:src && yarn run lint:prettier",
-    "lint:src": "yarn run lint:src:glob '**/*.{js,jsx,ts,tsx}'",
-    "lint:src:glob": "eslint --ignore-path .eslintignore",
-    "lint:prettier": "yarn run lint:prettier:glob '**/*.{yml,json,md,gql,graphql,flow}'",
-    "lint:prettier:glob": "prettier --ignore-path .eslintignore --list-different",
-    "fmt": "yarn run fmt:src && yarn run fmt:prettier",
-    "fmt:src": "yarn run fmt:src:glob '**/*.{js,jsx,ts,tsx}'",
-    "fmt:src:glob": "eslint --ignore-path .eslintignore --fix",
-    "fmt:prettier": "yarn run fmt:prettier:glob '**/*.{yml,json,md,gql,graphql,flow}'",
-    "fmt:prettier:glob": "prettier --ignore-path .eslintignore --write",
+    "build:types": "tsc --project tsconfig.declarations.json",
+    "build:transpile": "babel --extensions=.ts,.js,.jsx,.tsx src --out-dir lib --copy-files",
+    "lint": "yarn run lint:es",
+    "lint:es": "getk run lint-es",
+    "lint:fix": "yarn run lint:fix:es",
+    "lint:fix:es": "getk run fix-es",
     "prepublishOnly": "yarn run build",
     "postversion": "git push --follow-tags",
     "tdd": "yarn run test:mocha --watch",
     "test": "yarn run todos && yarn run lint && yarn run typecheck && yarn run test:mocha:coverage",
-    "test:mocha": "yarn run test:mocha:glob 'src/**/test.js'",
-    "test:mocha:coverage": "yarn run nyc --report-dir=.coverage --temp-directory=.coverage/.nyc_output --reporter=lcov --reporter=text-summary yarn run test:mocha",
-    "test:mocha:glob": "NODE_ENV=test yarn run mocha --opts mocha.opts",
+    "test:mocha": "yarn run test:mocha:glob '**/{,*.}test.ts'",
+    "test:mocha:coverage": "yarn run nyc --report-dir=coverage --temp-directory=coverage/.nyc_output --reporter=lcov --reporter=text-summary yarn run test:mocha",
+    "test:mocha:glob": "NODE_ENV=test yarn run mocha --require @babel/polyfill --require .register.js --extension ts",
     "todos": "yarn run todos:glob '**/*.{js,jsx,ts,tsx}'",
-    "todos:glob": "leasot --exit-nicely --tags '\\$FlowFixMe' --ignore='node_modules/**','lib/**','flow-typed/**'",
-    "typecheck": "flow"
-  },
-  "babel": {
-    "presets": [
-      "@babel/preset-env",
-      "@babel/preset-flow"
-    ],
-    "plugins": [
-      "@babel/plugin-transform-runtime"
-    ]
-  },
-  "eslintConfig": {
-    "parser": "babel-eslint",
-    "plugins": [
-      "goodeggs"
-    ],
-    "extends": [
-      "plugin:goodeggs/goodeggs",
-      "prettier",
-      "plugin:prettier/recommended"
-    ],
-    "env": {
-      "browser": true,
-      "node": true
-    },
-    "rules": {
-      "goodeggs/lodash-import-scope": "off"
-    }
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{js,jsx,ts,tsx}": [
-      "yarn run fmt:src:glob",
-      "git add"
-    ],
-    "*.{yml,json,md,gql,graphql,flow}": [
-      "yarn run fmt:prettier:glob",
-      "git add"
-    ]
```

Comment:
Looks like someone removed this configuration. We should either bring it back or remove it entirely (remove husky config file, remove husky and lint-staged dependencies).


This must be why it was logging warnings locally!